packages/renderer/src/docs/assay.vue

```diff
@@ -1,3 +1,181 @@
+<!-- TODO fix add buttons -->
+
 <template>
-  <h5>Assay Help</h5>
+  <div>
+  <h5>Assay</h5>
+
+    <div class="text-body2 text-justify">
+      In the Assay menu you can adapt the metadata of one assay added to your ARC.
+    </div>
+    <br>
+
+    <q-separator inset />
+
+    <div class="q-item row">
+      <div class="q-item__section column q-item__section--side justify-center">
+        <i class="q-icon text-grey-8 material-icons">biotech</i>
+      </div>
+      <div class="q-item__section column justify-center q-item__label">Assay</div>
+    </div>
+
+    <div class="q-pt-sm text-justify">
+      <q-list>
+              
+        <q-item>
+          <q-item-section>
+            <label class="q-field row no-wrap q-field--filled q-field--float q-field--dense" style="padding: 0.4em;"><!----><div class="relative-position col"><div class="q-field__control relative-position row no-wrap"><div class="col relative-position row no-wrap"><div class="q-field__label absolute">Identifier</div><!----></div></div><!----></div><!----></label>            
```

Comment:
These are kind of actual input fields with onhover effect? Is this by design? This would greatly increase the effort to correctly update the docs if changes in the button design arise. Maybe use images instead?

Reply:
This should be resolved somewhat in the future, for now we keep it.